### PR TITLE
Add maximum chunk size to data lake store docs.

### PR DESCRIPTION
**Pull request by jhendrixMSFT**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.

### CODE REVIEW

File: specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json [hunk 1]
```diff
@@ -64,7 +64,7 @@
               "format": "file"
             },
             "required": true,
-            "description": "The file contents to include when appending to the file."
+            "description": "The file contents to include when appending to the file.  The maximum content size is 4MB.  For content larger than 4MB you must append the content in 4MB chunks."
```
Comment (vladbarosan) [inline specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json]: where did we got this from ?

**Review by vladbarosan — APPROVED** [review]

File: specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json (hunk 1 above)
Comment (jhendrixMSFT) [reply specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json]: I've also asked somebody from the team to review.

File: specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json [hunk 2]
```diff
@@ -277,8 +277,10 @@
           }
         }
       }
-    },
-    "/webhdfs/v1/{destinationPath}": {
+    }
+  },
+  "x-ms-paths": {
+    "/webhdfs/v1/{destinationPath}?op=CONCAT": {
```
Comment (rahuldutta90) [inline specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json]: Is there a process to deprecate this Concat. We should only have MSConcat. Concat has some problem for which customer faces issues.

File: specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json [hunk 3]
```diff
@@ -395,7 +397,7 @@
         }
       }
     },
-    "/webhdfs/v1/{listFilePath}": {
+    "/webhdfs/v1/{listFilePath}?op=MSLISTSTATUS": {
```
[the remark this answers is not in the document]
Comment (jhendrixMSFT) [reply specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json]: This is for the older API version where the query param is defined as `MSLISTSTATUS`.  Is the query param wrong?

File: specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json [hunk 4]
```diff
@@ -509,7 +511,7 @@
         }
       }
     },
-    "/webhdfs/v1/{getFilePath}": {
+    "/webhdfs/v1/{getFilePath}?op=MSGETFILESTATUS": {
```
[the remark this answers is not in the document]
Comment (jhendrixMSFT) [reply specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json]: This is for the older API version where the query param is defined as `MSGETFILESTATUS`.  Is the query param wrong?

File: specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json (hunk 2 above)
Comment (jhendrixMSFT) [reply specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json]: Yes we can add `deprecated: true` and update the description string.  You want me to do that?

File: specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (rahuldutta90) [reply specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json]: It is not wrong, but it is deprecated.  So we are not updating the API Version as  a part of this PR. Then that is fine. The update in api version, is that done by the product team?

[the remark this answers is not in the document]
Comment (jhendrixMSFT) [reply specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json]: That's correct.  All I've done is added the additional documentation and fixed the build errors.

File: specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json (hunk 2 above)
Comment (rahuldutta90) [reply specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json]: Yes I think we should do that. Concat takes in input as queryparameters which causes another IIS filtering 404 error which is notoriously difficult to catch.

File: specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (rahuldutta90) [reply specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json]: No Action needed then.

File: specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (rahuldutta90) [reply specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json]: thanks, no action needed then.

**Comment by rahuldutta90** [comment]
@lewu-msft  Can you also please take a look at this?

File: specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json (hunk 2 above)
Comment (jhendrixMSFT) [reply specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json]: Done, see latest commit.

**Comment by lewu-msft** [comment]
LGTM

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

**Pull request merged**